templates/staff/user/role_list.html

```diff
@@ -51,20 +50,21 @@
             {% endfor %}
           {% endif %}
 
-          {% form_legend text="Assign roles site-wide (Note: this should usually only be used for internal staff users)" %}
+          {% form_legend text="Assign roles site-wide" class="hidden" %}
```

Comment:
Great spot. This should be `sr-only` instead of `hidden` – I'll tweak that now.

([source for `sr-only`](https://tailwindcss.com/docs/screen-readers))

Reply:
Ahhh that makes more sense, thanks!